tests/wicked/advanced/ref/t07_tun_interface_legacy.pm

```diff
@@ -25,7 +26,7 @@ sub run {
     record_info('Info', 'Create a TUN interface from legacy ifcfg files');
     $self->get_from_data('wicked/ifcfg/tun1_ref',      $config);
     $self->get_from_data('wicked/openvpn/server.conf', $openvpn_server);
-    assert_script_run("sed 's/device/tun1/' -i $openvpn_server");
+    sed_config($openvpn_server,{device => "tun1"});
```

Comment:
device without quotes and tun1 with quotes?

Reply:
It's perl hash syntax. If the key doesn't contains some special like a <space>, the quotes can be omitted.